stdlib/2and3/uuid.pyi

```diff
@@ -74,7 +94,7 @@ class UUID:
         def get_version(self) -> Optional[_Int]: ...
         def __cmp__(self, other: Any) -> _Int: ...
 
-def getnode() -> int: ...
+def getnode(*, getters: None = ...) -> int: ...  # getters is undocumented (and unused)
```

Comment:
Since it got removed from CPython master, maybe we should also just omit it from typeshed? There's no reason for anyone to actually use the argument.